compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -1675,6 +1675,24 @@ loco::NodeShape infer_unique(const luci::CircleUnique *node)
 }
 
 // Circle Only
+loco::NodeShape infer_lq_fully_connected(const luci::CircleLQFullyConnected *node)
+{
+  loco::TensorShape out_shape;
+  loco::TensorShape weights_shape;
+
+  auto input_shape = loco::shape_get(node->input()).as<loco::TensorShape>();
```

Comment:
Error occured here!

Reply:
haha, shape inference go Brrrrrrrrr
```suggestion
  auto input_shape = luci::shape_get(node->input()).as<loco::TensorShape>();
```

but now we have another problem. CircleChef produces messed up shapesm so circle2circle fails on verification.

Reply:
Or not.
Check recipe shapes and shape inference, please.
In detail:
- shape inference gives output shape [1, 32]
- recipe output shape is [32, 1]

Reply:
> haha, shape inference go Brrrrrrrrr
> 
> but now we have another problem. CircleChef produces messed up shapes, so circle2circle fails on verification.

Genius!

Reply:
> * recipe output shape is [32, 1]

Probably it's wrong output shape, we need to change it =)